tools/deployment_manager/configs/linux.json

```diff
@@ -87,14 +87,6 @@
         "runtime/3rdparty/hddl"
       ]
     },
-    "python3.6": {
-      "ui_name": "OpenVINO Python API for Python3.6",
-      "group": ["python"],
-      "dependencies" : ["ie_core"],
-      "files": [
-        "python/python3.6"
-      ]
-    },
```

Comment:
In general, I want to keep support of 3.6 version for open source users. Can we revert all places in code which prohibit to do it?

Reply:
Done, I reverted all asserts for Python version to be 3.7 back to 3.6, open source users will not encounter an assertion error when using 3.6.